starport/pkg/xgenny/xgenny.go

```
@@ -48,6 +51,11 @@ func (w Walker) walkDir(wl packd.WalkFunc, path string) error {
 
 		ppath := strings.TrimPrefix(path, w.trimPrefix)
 		ppath = filepath.Join(w.path, ppath)
+		if w.skip {
+			if _, err := os.Stat(ppath); os.IsExist(err) {
```

We should return the error if `err != nil && !os.IsExist(err)`